Engine/Code/Engine/Graphics/Buffers/FrameBuffer.cpp

```diff
@@ -30,14 +27,20 @@ void FrameBuffer::UnBind()
 void FrameBuffer::Invalidate()
 {
 	glGenFramebuffers(1, &m_FrameBufferID);
-	glBindFramebuffer(GL_FRAMEBUFFER, m_FrameBufferID);
 
 	glGenTextures(1, &m_ColorAttachment);
 	glBindTexture(GL_TEXTURE_2D, m_ColorAttachment);
 	glTexImage2D(GL_TEXTURE_2D, 0, GL_RGB8, 1024, 1024, 0, GL_RGB, GL_UNSIGNED_BYTE, NULL);
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
```

Comment:
@kevin30061994 Do you need to bind the sampler with a Frame buffer

Reply:
Nope but the texture params are definitely required. Samplers are only needed when trying to read these textures.

Reply:
Why does a function called Invalidate() initialize things?